Model/CustomerPaymentMethodsResult.php

```diff
@@ -0,0 +1,160 @@
+<?php
+namespace Paytrail\PaymentService\Model;
```

Comment:
Optional: please define namespaces as accurately as you can to avoid excessive implementations at the root of the Model directory. Remember that part of maintainability of your code is the ability for you or someone else to find the code that needs fixing after everyone has forgotten how it works.

Reply:
Done, moved Model files into Model/Token directory